package.json

```diff
@@ -38,13 +38,13 @@
     "karma-webpack": "^1.3.1",
     "lodash-node": "^2.4.1",
     "mocha": "^2.0.1",
-    "react-addons-test-utils": "^0.14.6",
+    "react-addons-test-utils": "15.0.x",
     "webpack": "^1.12.2",
     "webpack-dev-server": "^1.12.0"
   },
   "peerDependencies": {
-    "react": "0.14.8",
-    "react-dom": "^0.14.0"
+    "react": "15.0.x",
```

Comment:
after this is merged, we will bump the version to 2.1.0. Our app locks this at 2.0.0, which will forever remain in its current state via npm.